src/backup/background/index.ts

```diff
@@ -9,6 +9,7 @@ import estimateBackupSize from './estimate-backup-size'
 import BackupProcedure from './procedures/backup'
 import { BackupRestoreProcedure } from './procedures/restore'
 import { ProcedureUiCommunication } from 'src/backup/background/procedures/ui-communication'
+import NotificationBackground from '../../notifications/background'
```

Comment:
More of a quick tip for the future than something urgent to change: we set up TS to be able to import using absolute paths from `src/` dir, so it's probably a lot easier to write and understand to just do `import 'src/notifications/background'` for imports that reach into other feature modules (top-level directories).